spec/javascripts/live_search_spec.js

```diff
@@ -33,7 +33,7 @@ describe('liveSearch', function () {
     search_results: '<div class="finder-results js-finder-results" data-module="gem-track-click">' +
       '<ol class="gem-c-document-list">' +
         '<li class="gem-c-document-list__item">' +
-          '<a data-track-category="navFinderLinkClicked" data-track-action="" data-track-label="" class="gem-c-document-list__item-title" href="aaib-reports/test-report">Test report</a>' +
+          '<a data-track-category="navFinderLinkClicked" class="gem-c-document-list__item-title" href="aaib-reports/test-report">Test report</a>' +
```

Comment:
Thanks - I actually did remove all the `data-track-category` references in a future commit - as `data-track-action` was the first thing I began to remove, I was trying to keep the commits of things I removed separated to avoid enormous commits